src/coreclr/jit/lowerxarch.cpp

```diff
@@ -4052,6 +4052,7 @@ GenTree* Lowering::LowerHWIntrinsicTernaryLogic(GenTreeHWIntrinsic* node)
                             replacementNode = comp->gtNewAllBitsSetConNode(simdType);
                         }
 
+                        BlockRange().Remove(op4);
```

Comment:
This should be removable for the `TernaryLogicUseFlags::C` path as well, so can likely be removed alongside `node` if `replacementNode != nullptr`